.gitignore

```diff
@@ -9,4 +9,3 @@
 .DS_Store
 *.log
 .idea
-.vscode
```

Comment:
I'd thought I'd check in the vscode project settings so future contributors using vscode would have the correct style settings for this repro.  It shouldn't do any harm having those settings checked in.